### PR TITLE
Clarify syntax in pylint.lint example

### PR DESCRIPTION
**Pull request by felixvd**
This makes the call in the documentation non-trivial, so that [idiots like me](https://stackoverflow.com/questions/69851634/how-to-run-pylint-from-inside-python/69851635#69851635) don't get stuck.

If someone could show how to get the output of the command (as asked [here](https://stackoverflow.com/questions/56081299/how-to-get-report-score-from-pylint-when-run-python-code)), that would also be very helpful to add.

- [ ] Add yourself to CONTRIBUTORS if you are a new contributor.
- [ ] Add a ChangeLog entry describing what your PR does.
- [ ] If it's a new feature, or an important bug fix, add a What's New entry in
      `doc/whatsnew/<current release.rst>`.
- [ ] Write a good description on what the PR does.

### CODE REVIEW

**Review by Pierre-Sassoulas — CHANGES_REQUESTED** [review]
Thank you for the better example ! I think your commit message `so idiots don't me get stuck.` could be reordered to `so it's harder to get stuck.` More seriously could you add yourself to the contributors, please ? :)

I'll keep the stackoverflow question in mind, I did not manage to make it work myself yet, which is a clue that new documentation will be required.

**Comment by felixvd** [comment]
Done. I assumed it's not worth making an entry in the file for such a small contribution, but if it's policy then sure.

**Comment by DanielNoord** [comment]
> I'll keep the stackoverflow question in mind, I did not manage to make it work myself yet, which is a clue that new documentation will be required.

I have investigated this a bit more. Without changes to `Run` I think this is simply impossible. The way we change `.out` and `._output` doesn't really allow to do this with `Run`.
When using `PyLinter` this might be possible, but I'm not sure we want to create documentation telling people to access it directly. In any case, we could open an issue to track this and close this PR (waiting for a change to `Run` seems incorrect).

**Comment by Pierre-Sassoulas** [comment]
> Without changes to Run I think this is simply impossible.

That was my conclusion too. I had the impression it was a problem with kwargs not being taken into account, but I did not go in depth. Do you want to create the issue @DanielNoord ? 

>  I'm not sure we want to create documentation telling people to access it directly

I pretty much agree, I think we need ``Run`` to be the entry-point. Especially as we will want to make change to Pylinter later as it's so bloated.

**Comment by DanielNoord** [comment]
Opened an issue to track the `output` issue :)

File: CONTRIBUTORS.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿Contributors
+Contributors
```
Comment (Pierre-Sassoulas) [inline CONTRIBUTORS.txt:1]:
```suggestion
Contributors
```

**Review by Pierre-Sassoulas — APPROVED** [review]
I'm merging as this is still clearer than what we had before. Follow up to be able to recover the output in a stream is #5295.

**Pull request merged**